src/Symfony/Component/DependencyInjection/Compiler/Compiler.php

```diff
@@ -95,6 +102,8 @@ public function addPass(CompilerPassInterface $pass, $type = PassConfig::TYPE_BE
      */
     public function addLogMessage($string)
     {
+        @trigger_error($string);
```

Comment:
looking at the phdoc, the default is `E_USER_NOTICE`, so all good.
but I now made it explicit, and configurable (via a third "$level" arg)